article/_posts/2020-11-19-AlphaMuGoZero.md

```diff
@@ -0,0 +1,181 @@
+
+$$l = (z - v)^2 - \pi^T \log p + c \|\theta\|^2$$
+
+where $$c$$ in this case is a regularization term, is used to train the network. We can interpret the loss as cross-entropy forcing the policy to match MCTS' behavior.
+
+![](/article/images/alpha/fig5.jpg)
+> Silver, D., Schrittwieser, J., Simonyan, K. et al. Mastering the game of Go without human knowledge. Nature 550, 354–359 (2017). https://doi.org/10.1038/nature24270
+
+From the above figure, we can see that the algorithm can surpass AlphaGo after only ~40h of training. Interestingly enough, we can see that AlphaGo Zero is not really good at prediction moves, but quite good at predicting game outcome. A supervised learning version of the algorithm is shown for comparison. Because it manages to outperform AlphaGo (and therefore the best human player is ELO is transitive), the results seem to imply that AlphaGo Zero learns to play differently than human players (which could explain moves 37 and 78). 
+## AlphaZero
+
+Still on a quest to remove priors, Deepmind then released AlphaZero. AlphaZero keeps the same model and overall training process as AlphaGo Zero but removes some components that do not translate well to other games. Notable changes include:
+
+- Removal of data augmentation (rotating and flipping boards) that was done by AlphaGo Zero because it would create impossible configurations for Chess and Shogi
+- Opponents for self-play are now simply the exact same neural network instead of a roster of previous versions of the algorithm
+- AlphaZero reuses the same hyperparameters but one (exploration is dependent on the number of valid moves) for all games 
+
+![](/article/images/alpha/tab3.jpg)
+![](/article/images/alpha/tab4.jpg)
+> Silver, David, et al. “Mastering Chess and Shogi by Self-Play with a General Reinforcement Learning Algorithm.” ArXiv:1712.01815 [Cs], Dec. 2017. arXiv.org, http://arxiv.org/abs/1712.01815
+
+Handcrafted features make a return, as shown above, and action probabilities now must take a different form, also shown above. 
+
+![](/article/images/alpha/fig6.jpg)
+![](/article/images/alpha/tab5.jpg)
+> Silver, David, et al. “Mastering Chess and Shogi by Self-Play with a General Reinforcement Learning Algorithm.” ArXiv:1712.01815 [Cs], Dec. 2017. arXiv.org, http://arxiv.org/abs/1712.01815
+
+Above are shown results from the training process. We can observe that AlphaZero manages to defeat both AlphaGo Zero as well as game-specific algorithms.
+
+## MuZero
+
+Chess, Shogi and Go are games that can easily offer a perfect simulator as their rules are simple and well defined. However, this is not the case for most real-world scenarios such as robotics. Model-based reinforcement learning tries to tackle this problem by learning a simulator and then allowing planning using this learned simulator.
```

Comment:
Since the example given is about Atari games and not robotics, may be this sentence
(...) most real-world scenarios such as robotics.
can be re-worded as
(...) most real-world scenarios, such as robotics or other continuous-space games and problems.

Reply:
Fixed ! 137ef52